brave/brave-core
# Add backup notice

Resolves https://github.com/brave/brave-browser/issues/10021

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

- Clean profile
- Enable Rewards
- Restore wallet with user funds
- Wait for `OnFetch` callback
- Click on gear to bring up backup/restore modal
- "Backup" tab should be selected by default
- Verify that modal text matches spec
- Verify that clicking the "connect to Uphold" link initiates the Uphold connection process
- Verify that you can switch back and forth between Backup/Restore tabs
-----
- Clean profile
- Enable Rewards
- Claim promotion
- Wait for `OnFetch` callback
- Click on gear to bring up backup/restore modal
- "Restore" tab should be selected by default
- Should be unable to switch to "Backup" tab; instead, clicking "Backup" will display a modal notice
- Verify that clicking the "connect to Uphold" link in the modal notice initiates the Uphold connection process

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
